apps/onboarding/src/app.controller.ts

```diff
@@ -281,4 +275,34 @@ export class AppController {
       txHash: resp.payload
     }
   }
+
+  private logMetaTransaction(
+    relayerResp: RelayerResponse<string>,
+    metaTx: RawTransaction,
+    childTxs: RawTransaction[],
+    session: SessionEntity
+  ) {
+    this.logger.event(EventType.MetaTransactionSubmitted, {
```

Comment:
Should we also log the MTW nonce?

Reply:
We don't really have it here, we can decode it from the payload but yeah it might be helpful to debug some issues.

Reply:
ah nvm we actually don't have it and can't determine it because at this point the nonce is only encoded in the signature